### ISSUE TITLE
TOOLTIP: IDS-Tooltip/Tooltip Attribute Is Not Working For IDS Icon.

### ISSUE THREAD

**Issue by ashachodavarapu** [infor-design/enterprise-wc#2257]
**Issue**
When I add the tooltip attribute to the IDs icon, it doesn't work as expected. Additionally, when I try to use the
<ids-tooltip>
tag, I can see the tooltip, but it behaves differently, such as flickering and appearing in different positions when I hover over different items. Please refer below video.
**Steps to reproduce the behavior:**
- Add the below code in html and observe the issue in UI

` <ids-icon id="priority" icon="exclamation"></ids-icon>
                <ids-tooltip target="#priority" id="priority-tooltip">{{ "HIGH_PRIORITY" | translate }}</ids-tooltip>`

**Expected behavior**
- The tooltip should work for the IDs icon and be correctly positioned. For the <ids-tooltip> tag, it should also display in the correct position without flickering

**Version**
- ids-enterprise:^1.0.0

**Screenshots**
https://github.com/infor-design/enterprise-wc/assets/105051938/001db420-3999-45e0-bca9-37875292aa0d

**Comment by tmcconechy** [comment]
I think we dont want to add this. The icon is used everywhere and adding all the tooltip stuff with it may make it too "heavy". I would rather see if we can get your case working through either ids-alert or ids-button.

-Tim

**Comment by ashachodavarapu** [comment]
@tmcconechy, I found if I use  appearance="tertiary-destructive" for the button icon, it's changed and the tooltip also works fine.

**Comment by tmcconechy** [comment]
@ashachodavarapu ok great. That is a good idea. For this issue i'll just review if we should add tooltips on ids-icon and decide if we close or not

**Comment by ashachodavarapu** [comment]
@tmcconechy 
yeah, I am using unique IDs and iterating the list of items(using this icon inside the list item).
By using this example I tried=>https://main.wc.design.infor.com/ids-icon/tooltip.html

**Comment by jdilag90** [comment]
@ashachodavarapu 

I created an example similar to yours in our angular example using wc 1.4
It takes text from a component variable and is put through an ng uppercase pipe 
<img width="953" alt="image" src="https://github.com/user-attachments/assets/136f2348-24a3-440f-a09f-aa8c86d7385e">

Here's clip of the result and it seems to be working.
https://github.com/user-attachments/assets/b6261f5e-f6ed-4e29-8fd0-60537bf6e36c

Is there any other context to your use case that you can provide to help us recreate the issue you're experiencing?

**Comment by ashachodavarapu** [comment]
@jdilag90 
I am using like this inside the div 
`  @if (item.priority === "HIGH") {
                <ids-icon id="high-priority" icon="exclamation"></ids-icon>
                <ids-tooltip target="#high-priority" id="high-priority-tooltip">{{
                    "HIGH_PRIORITY" | translate
                }}</ids-tooltip>
            }`
            
and this component html I am using in the list view
`  <ids-list-view-item><nc-item></nc-item></ids-list-view-item>`

Output:
![image](https://github.com/user-attachments/assets/474249ec-a5ae-41a8-9c27-7de0c644aaf7)
sometimes I am unable to see the tooltip
and also I can see the half-tooltip text only.
